go/signedexchange/README.md

```diff
@@ -1,3 +1,83 @@
-# gen-signedexchange
+# go/bundle
+This directory contains a reference implementation of [Signed HTTP Exchanges](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html) spec.
 
-This directory will hold a reference implementation of a tool to sign exchanges, but for now, the version that best matches Chrome is in https://github.com/nyaxt/webpackage/tree/master/go/signedexchange .
+## Overview
+We currently provide two command-line tools: `gen-signedexchange` and `gen-certurl`.
+
+`gen-signedexchange` generates a signed exchange file. The `gen-signedexchange` command constructs HTTP request and response pair from given command line flags, attach a cryptographic signature of the pair, and serializes the result to output file.
+
+`gen-certurl` converts a X.509 certificate chain to `application/cert-chain+cbor` format, which is defined in the [Section 3.3 of the Signed HTTP Exchanges spec](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#rfc.section.3.3).
```

Comment:
Since `gen-certurl` accepts an OCSP response and SCT, I'd say 

> ... converts an X.509 certificate chain, an OCSP response, and an SCT (if one isn't already included in the certificate or OCSP response) to `application/cert-chain+cbor` ...

Reply:
Taken